circuits/transfer/src/execute/output.rs

```diff
@@ -32,7 +32,7 @@ impl CircuitOutput {
     /// Zeroed stealth address used for padding
     pub const ZERO_STEALTH_ADDRESS: StealthAddress = {
         let pk = Self::ZERO_COMMITMENT;
-        let pk = PublicKey::from_raw_unchecked(pk);
+        let pk = NotePublicKey::from_raw_unchecked(pk);
```

Comment:
Same, use npk or note_pk here

Reply:
I was going to change the variable names when also changing the method names, but I might as well do it now

Reply:
As you prefer then! I approve.